.github/workflows/ansible-test.yml

```diff
@@ -0,0 +1,59 @@
+# README FIRST
+# 1. replace "NAMESPACE/COLLECTION_NAME" with the correct name, ie "community/zabbix"
+# 2. If you don't have unit tests remove that section
+# 3. If your collection depends on other collections ensure they are installed, see "Install collection dependencies"
+# If you need help please ask in #ansible-devel on Freenode IRC
+
+name: CI
+on:
+# Run CI against all pushes (direct commits) and Pull Requests
+- push
+- pull_request
+
+jobs:
+
+###
+# Sanity tests (REQUIRED)
+#
+# https://docs.ansible.com/ansible/latest/dev_guide/testing_sanity.html
+
+  sanity:
+    name: Sanity (Ⓐ${{ matrix.ansible }}+py${{ matrix.python }})
```

Comment:
UTF char in string - I'm guessing we really don't want that.

Reply:
Removed.